src/vs/workbench/contrib/comments/browser/commentThreadWidget.ts

```diff
@@ -409,11 +409,6 @@ export class CommentThreadWidget<T extends IRange | ICellRange = IRange> extends
 			content.push(`.review-widget .body .review-comment blockquote { background: ${blockQuoteBackground}; }`);
 		}
 
-		const blockQuoteBOrder = theme.getColor(textBlockQuoteBorder);
```

Comment:
I forgot all this css was here. I should move it into a css file. Opened https://github.com/microsoft/vscode/issues/272240 to track moving it.